Formula/s/staq.rb

```diff
@@ -18,7 +18,7 @@ class Staq < Formula
   depends_on "cmake" => :build
 
   def install
-    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "-S", ".", "-B", "build", "-D", "INSTALL_SOURCES=ON", *std_cmake_args
```

Comment:
Why do we want this?

Reply:
@p-linnane so one can use staq's API in own code. If that's not OK for formulas I'll remove it.

Reply:
Works for me. Thanks!